lab_1/main.py

```diff
@@ -4,68 +4,135 @@
 """
 
 
-def tokenize(text: str) -> list or None:
+import json
+
+from os.path import exists
+from typing import Optional
+
+
+def tokenize(text: str) -> Optional[list]:
     """
     Splits a text into tokens, converts the tokens into lowercase,
     removes punctuation and other symbols from words
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    if isinstance(text, str):
+        tokenized = (
+            "".join([char for char in text if char.isalpha() or char.isspace()])
+            .strip()
+            .lower()
+            .split()
+        )
+        return tokenized
+    return None
```

Comment:
put this check to the beginning
If the argument is incorrect, return `None` just there.
This way you won't have numerous `return None` afterwards